apps/blog/src/posts/data-insights-on-PWAs/data-insights.md

```diff
@@ -0,0 +1,61 @@
+
+### Manifests and app capabilities
+
+Manifests are a hard requirement for a PWA, unlike service workers. They hold a lot of crucial information that is needed for the PWA to behave more app-like such as icons, file handlers, link handlers and so much more. 
+
+ ![Alt text](/posts/data-insights-on-PWAs/SW.png)
+
+ We took 6 of the app capabilities into account to try and analyze how often they are being used in PWAs these days. A quick description of these:
+
+ * File handlers:  In a PWA, the file_handlers member specifies the types of files the PWA can handle. When installed, the browser uses this to associate the PWA with specific file types at the OS level.
+ * Launch handlers: In a PWA, the launch_handler field specifies how the PWA should be launched and navigated to. 
+ * Protocol handlers: In a PWA, protocol handlers are used to register the PWA to handle certain allowed protocols. A protocol is a string that allows clients to identify a resource on the web, such as http or mailto. When a PWA is installed, it can register to handle specific protocols, such as mailto or geo.
+ * Share target:  A share target is a feature in a Progressive Web App (PWA) that allows the app to receive shared content from other apps. This is done by adding a share_target member to the web app manifest, which specifies how the app should handle shared data. It allows for seamless integreation of the PWA with the native sharing capabilities of the device. 
+ * Shortcuts: Shortcuts in a Progressive Web App (PWA) allow users to quickly access specific features or pages within the app. These shortcuts can be added to the web app manifest and are displayed when a user long-presses or right-clicks on the app icon on their device.
+ * Widgets: Widgets are small applications that can be embedded within a larger application or operating system. They are designed to provide quick access to frequently used information or functionality. In the context of an app manifest, widgets are defined in the PWA manifest file using the widgets manifest member.
+
+## Insights from the data
+
+* It seems like  shortcuts are among the more popular features that developers like to use. Shortcuts are relatively easy to add, and are a quick way (or should a say, a shortcut), to add a feature that makes the PWA more app like.
```

Comment:
I liked that you mentioned that shortcuts is easy to add but it also may be good to mention that features like widgets (below) or share_target require multiple steps and aren't as easy to add as well.